website/themes/uppy/layout/example.ejs

```diff
@@ -51,8 +51,11 @@
       };
     }
 
+    // IE 10 doesn’t support console.debug
+    var consoleDebug = console.debug || console.log
+
     console.log = customLog(console.log.bind(console), document.getElementById("console-log"));
```

Comment:
I guess we don’t need `console.log =` here then, because `.debug` calls `.log` when needed? But it’s a minor detail, it’s not hurting anything.